mobile/docs/TECHNICAL_GUIDE.md

```diff
@@ -0,0 +1,457 @@
+### Environment Configuration
+
+#### Android Emulator
+```dart
+// lib/services/api_config.dart
+static String _baseUrl = 'http://10.0.2.2:3000';
+```
+
+#### iOS Simulator
+```dart
+static String _baseUrl = 'http://localhost:3000';
+```
+
+#### Physical Device
+```dart
+static String _baseUrl = 'http://YOUR_COMPUTER_IP:3000';
+// Or use production URL
+static String _baseUrl = 'https://your-domain.com';
+```
+
+### Common Commands
+
+```bash
+# Install dependencies
+flutter pub get
+
+# Run app
+flutter run
+
+# Build APK
+flutter build apk --release
+
+# Build App Bundle
+flutter build appbundle --release
+
+# Build iOS
+flutter build ios --release
+
+# Code analysis
+flutter analyze
+
+# Run tests
+flutter test
+```
+
+### Debugging Tips
+
+1. **View Network Requests**:
+   - Android Studio: Use Network Profiler
+   - Or add `print()` statements in code
+
+2. **View Stored Data**:
+   ```dart
+   // Add at debugging point
+   final prefs = await SharedPreferences.getInstance();
+   print('Backend URL: ${prefs.getString('backend_url')}');
+   ```
+
+3. **Clear Local Data**:
+   ```bash
+   # Android
+   adb shell pm clear com.example.sure_mobile
+
+   # iOS Simulator
+   # Long press app icon -> Delete app -> Reinstall
+   ```
+
+## CI/CD
+
+The project is configured with GitHub Actions automated builds:
+
+### Trigger Conditions
+- Push to `main` branch
+- Pull Request to `main` branch
+- Only triggers when Flutter-related files change
+
+### Build Process
+1. Code analysis (`flutter analyze`)
+2. Run tests (`flutter test`)
+3. Android Release build (APK + AAB)
+4. iOS Release build (unsigned)
+5. Upload build artifacts
+
+### Download Build Artifacts
+Available on GitHub Actions page:
+- `app-release-apk`: Android APK file
+- `app-release-aab`: Android App Bundle (for Google Play)
+- `ios-build-unsigned`: iOS app bundle (requires signing for distribution)
+
+## Future Extensions
+
+### Planned Features
+- **Transaction History**: View and manage transaction history
+- **Account Sync**: Support automatic bank account synchronization
+- **Budget Management**: Set and track budgets
+- **Investment Tracking**: View investment returns
+- **AI Assistant**: Financial advice and analysis
+- **Push Notifications**: Transaction alerts and account change notifications
+- **Biometric Authentication**: Fingerprint/Face ID quick login
+- **Multi-language Support**: Chinese/English interface switching
+- **Chart Analysis**: Financial data visualization
+
+### Technical Improvements
+- Offline mode support
+- Data caching optimization
+- More robust error handling
+- Unit tests and integration tests
+- Performance optimization
+
+## License
+
+This project is distributed under the AGPLv3 license.
+
+## Contributing
+
+Issues and Pull Requests are welcome!
+
+## Related Links
+
+- **Backend Project**: https://github.com/we-promise/sure
+- **Flutter Official Documentation**: https://docs.flutter.dev
+- **Dart Language Documentation**: https://dart.dev/guides
```

Comment:
I believe we can skip these, no?

Reply:
Agree.  This is probably carryover before the move to the monorepo.